bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryClient.java

```diff
@@ -219,28 +236,27 @@ public boolean deleteTable(TableId tableId) {
    */
   public Job overwriteDestinationWithTemporary(
       TableId temporaryTableId, TableId destinationTableId) {
-    String queryFormat =
```

Comment:
Yes, this approach of avoiding the use of DML means we no longer have to wait for the temp file streaming buffer to be flushed before it can be copied to the destination table.